device/rg28xx/script/control.sh

```diff
@@ -46,3 +46,16 @@ for file in "$DEVICE_CONTROL_DIR/openbor/"*.ini; do
         cp "$file" "$DC_STO_ROM_MOUNT/MUOS/emulator/openbor/userdata/system/configs/openbor/"
     fi
 done
+
+# Set GBA SP Overlay as default in gpSP / mGBA
+GP_CFG="/$DC_STO_ROM_MOUNT/MUOS/info/config/gpSP/gpSP.cfg"
```

Comment:
No requirement to have a `/` at the start as those variables pull from the device configuration which start with a `/` already.  We also might want to use the global storage preference variable for configurations here (`$GC_STO_CONFIG`) and ensure the script is initialised if it is not already `. /opt/muos/script/var/global/storage.sh`

Reply:
We'll probably need to update the control scripts for all the other devices too with the aforementioned variable